DESCRIPTION

```diff
@@ -41,6 +41,7 @@ Depends:
     teal.slice (>= 0.5.1.9009)
 Imports:
     checkmate (>= 2.1.0),
+    crayon,
```

Comment:
Is this just for the colouring of the output? Do you think we could live without it? Not a big deal, but always an extra dependency

Reply:
Alrighty, just checkin